addons/account_facturx/wizard/account_invoice_send.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
+
+
+class AccountInvoiceSend(models.TransientModel):
+    _inherit = 'account.invoice.send'
+    _description = 'Account Invoice Send'
+
+    xml_type = fields.Selection(selection_add=[('facturx', 'Factur-X')])
```

Comment:
I don't think this is a good idea. You probably want to manage 2 flows:
- the xml is embedded inside the pdf: do nothing
- the xml is not embedded inside the pdf: send the pdf + the xml to your customer.